### ISSUE TITLE
[FE] Profile 2.0: Complete final accessibility review tests

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#15707]
## Background

As part of the final accessibility & 508 office review (#15247), we need to re-do some of the accessibility tests we did before launch:

- [Axe scans in daily development](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#axe-scans-in-daily-development)
- [Screen reader tests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#screen-reader-tests)

You can find the original results for these tests in #11273.

## Tasks

- [x] Complete axe scans in daily development
- [x] Complete screen reader tests
- [ ] Create tickets if issues are found

**Comment by jenstrickland** [comment]
Thank you! I love how you broke the work up into manageable chunks. 

The team doesn't need to share any results of their tests with me. They should just write issue tickets for anything found, and provide the list in the full audit request, and check all the checkboxes for the tests they complete.

**Comment by Samara-Strauss** [comment]
Noted, thank you!

**Comment by sanlouise** [comment]
Running the screen reader test again, I did not find issues that should be addressed by the frontend team. I still believe that it is an awkward workflow when the user is editing one field, and then tries to edit another field - but I addressed this with Jennifer in [the results of the previous tests](https://github.com/department-of-veterans-affairs/va.gov-team/issues/11273).

Running the axe scans, we got one 'best practice' suggestion for when we display the profile-wide alert (which contains an h3) above the profile content, which contains an h2. I don't think there is anything we can do to resolve this, though - and it was not marked as an error.

![image](https://user-images.githubusercontent.com/14869324/98389786-c5343d00-2011-11eb-9a94-302c07ef80f9.png)


@Samara-Strauss I don't think there are any other action items on my end.

**Comment by Samara-Strauss** [comment]
Thank you for letting me know! @jenstrickland looks like everything is good here. I'm going to close this ticket.

**Comment by jenstrickland** [comment]
@sanlouise Thanks for checking on this. That alertbox heading should have the `level` prop value changed to `2` to address the heading order issue ([alertbox developer documentation](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/alertbox/).  Please let me know if I may help.

**Comment by Samara-Strauss** [comment]
Closing this again. Just created #15908 for the fix Jen mentioned.
